servo:
    hardware_interface:
        loop_hz: 50 # Hz
        type: servo_hardware_interface/ServoHW
        joints:
            - a1_joint
            - a2_joint
            - a3_joint
            - a4_joint
            - a5_joint
            - a6_joint

